lib/acme/client/resources/challenges/dns01.rb

```diff
@@ -12,6 +12,6 @@ def record_type
   end
 
   def record_content
-    crypto.digest.hexdigest(authorization_key)
+    Base64.urlsafe_encode64(crypto.digest.digest(authorization_key)).sub(/[\s=]*\z/, '')
```

Comment:
It drops the padding `urlsafe_encode64` adds, which Ruby's stdlib sadly has no option for.